.github/workflows/deploy-to-openshift-frontend-efx.yml

```diff
@@ -0,0 +1,159 @@
+name: 4 EFX - Deploy Frontend
+
+env:
+  # 🖊️ EDIT your repository secrets to log into your OpenShift cluster and set up the context.
+  # See https://github.com/redhat-actions/oc-login#readme for how to retrieve these values.
+  # To get a permanent token, refer to https://github.com/redhat-actions/oc-login/wiki/Using-a-Service-Account-for-GitHub-Actions
+  OPENSHIFT_SERVER: ${{ secrets.OPENSHIFT_SERVER }}
+  OPENSHIFT_TOKEN: ${{ secrets.OPENSHIFT_TOKEN }}
+  # 🖊️ EDIT to set the kube context's namespace after login. Leave blank to use your user's default namespace.
+  OPENSHIFT_NAMESPACE: ${{ secrets.OFM_NAMESPACE_NO_ENV }}-test
+
+  # SPLUNK_TOKEN: ${{ secrets.SPLUNK_TOKEN }}
+
+  # 🖊️ EDIT to change the image registry settings.
+  # Registries such as GHCR, Quay.io, and Docker Hub are supported.
+  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  IMAGE_REGISTRY_USER: ${{ github.actor }}
+  IMAGE_REGISTRY_PASSWORD: ${{ github.token }}
+
+  APP_NAME: 'ofm'
+  REPO_NAME: 'ecc-ofm'
+  #grabs the branch name from github dynamically
+  BRANCH: ${{ github.ref_name }}
+  IMAGE_NAME: 'ecc-ofm-frontend'
+  APP_NAME_FRONTEND: 'frontend'
+  # We have multiple environments in each openshift namespace dev, test in DEV and uat, efx in TEST. This setting will help name the deployment config and routes for each.
+  APP_ENVIRONMENT: 'efx'
+
+  NAMESPACE: ${{ secrets.OFM_NAMESPACE_NO_ENV }}
+
+  MIN_REPLICAS: '2'
+  MAX_REPLICAS: '3'
+  MIN_CPU: '50m'
+  MAX_CPU: '100m'
+  MIN_MEM: '200Mi'
+  MAX_MEM: '250Mi'
+  # SITE_URL should have no scheme or port. It will be prepended with https://
+  HOST_ROUTE: ${{ secrets.SITE_URL }}
+  CA_CERT: ${{ secrets.CA_CERT }}
+  CERTIFICATE: ${{ secrets.CERTIFICATE }}
+  PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+
+  # Frontend configuration
+  BANNER_ENVIRONMENT: 'EFX'
+  BANNER_COLOR: '#800080'
```

Comment:
I have no idea where the banner colors came from but we need a new one. Maybe a darker green (Dev is light green) as blue/yellow conflict with the palette and red screams error.

Reply:
@weskubo-cgi 
![image](https://github.com/user-attachments/assets/eadab1dd-1bed-492c-a234-3a6ed9cdb354)

How's this?